library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -446,17 +447,24 @@ public void pauseBatch(long id) {
     }
 
     public void resumeBatch(long id) {
-        ContentValues values = new ContentValues();
-        values.put(DownloadContract.Downloads.COLUMN_CONTROL, DownloadsControl.CONTROL_RUN);
-        values.put(DownloadContract.Downloads.COLUMN_STATUS, DownloadStatus.PENDING);
+        ContentValues values = getResumeContentValues();
         String where = COLUMN_BATCH_ID + "= ? AND " + DownloadContract.Downloads.COLUMN_STATUS + " != ?";
-        String[] selectionArgs = {String.valueOf(id), String.valueOf(DownloadStatus.PENDING)};
+        String[] selectionArgs = {String.valueOf(id), String.valueOf(DownloadStatus.SUCCESS)};
```

Comment:
Doesn't really matter if they're pending and you set them to pending again.

Also @zegnus commit message says to update non-complete items only but it updated non-pending items - was this intentional?